setup.cfg

```diff
@@ -40,16 +40,16 @@ project_urls =
 packages = find:
 package_dir =
     =src
-python_requires = ~=3.8
+python_requires = >=3.8
 install_requires =
     click >= 7.0
     click-loglevel ~= 0.2
     in_place ~= 0.4
     pydantic ~= 1.7
     PyGithub ~= 1.55
     python-dateutil ~= 2.7
-    python-dotenv ~= 0.11
-    PyYAML ~= 5.0
+    python-dotenv ~= 1.0
+    PyYAML >= 5.0
```

Comment:
if we are still compatible with prior ones, should we still allow them too?     is there an easy way to spec that?

Reply:
Adjusted.